website_sale_qty/__openerp__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Website Sale - Price Tiers',
+    'summary': 'Add price tier radio buttons to product pages in website shop',
+    'version': '9.0.1.0.0',
+    'category': 'E-commerce',
+    'website': 'https://laslabs.com/',
+    'author': 'LasLabs',
+    'license': 'AGPL-3',
```

Comment:
Please switch all of this to LGPL-3. Get with me in chat when you have some time so we can discuss licensing